src/ITensorMPS/abstractprojmpo/projmpo_mps2.jl

```diff
@@ -0,0 +1,46 @@
+using ITensors: contract
```

Comment:
```suggestion
using ITensors: ITensors, contract
```
since we are overloading `contract` with `ITensors.contract`.